components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIDefinitionFromSwagger20.java

```diff
@@ -208,6 +210,21 @@ private void populateConfigMapForScopes(Swagger swagger, String namespace) {
     }
 
     @Override
+    public Map<String, String> getScope(String resourceConfigJSON) throws APIManagementException {
```

Comment:
Not yet. After finalizing the yaml files we have to keep one getScope(resourceConfigJson) file and write tests.